### DISCUSSION TITLE
Remove a system from Dmarc whitelist

### DISCUSSION THREAD

**Discussion by flaviozluca** [rspamd/rspamd#3825] [Q&A]
Friends! How can I suggest team to remove a domain from the default DMARC Whitelist?

Thanks,.

**Comment by vstakhov** [comment]
You can open a pull request for https://github.com/rspamd/maps/ repo

**Reply by vstakhov** [reply]
Specifically for https://github.com/rspamd/maps/blob/master/rspamd/dmarc_whitelist_new.inc file

**Comment by flaviozluca** [comment]
Thanks!
